python/python/glide/__init__.py

```diff
@@ -28,10 +28,6 @@
     FlushMode,
     InfoSection,
     InsertPosition,
-    StreamAddOptions,
```

Comment:
I moved these from core.py to a new stream.py file since I was creating more stream classes and it seems to make more sense. We also have the bitmap.py and sorted_set.py files that hold classes related to those groups of commands.